lib/task-data/base-tasks/wait-sel-events.js

```diff
@@ -0,0 +1,12 @@
+// Copyright 2017, Dell EMC, Inc.
+
+'use strict';
+
+module.exports = {
+    friendlyName: 'Wait for SEL events',
+    injectableName: 'Task.Base.Wait.Sel.Events',
+    runJob: 'Job.Wait.Sel.Events',
+    requiredOptions: ['alertFilters'],
```

Comment:
Meanwhile, if the optionsSchema is specified, the `requiredOptions` could be omitted.